src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAO.java

```diff
@@ -49,13 +41,7 @@ public boolean save(MediaEntry mediaEntry) throws SQLException, IOException, Cla
      */
     @Override
     public MediaEntry fetch(int id) throws SQLException, IOException, ClassNotFoundException {
-        addWhere("id", id);
-        List<MediaEntry> entries = parse(select());
-
-        if(entries.isEmpty())
-            return null;
-
-        return entries.get(0);
+        return allMediaEntries.get(id);
```

Comment:
This is an example of extra code caused by writing your own implementation of CrudRepository